store/kv/kv.go

```diff
@@ -114,6 +249,16 @@ func (k *KeyValueStoreImpl) ReadRange(ctx context.Context, table []byte, lkey Ke
 	}, nil
 }
 
+func (m *KeyValueStoreImplWithMetrics) ReadRange(ctx context.Context, table []byte, lkey Key, rkey Key) (Iterator, error) {
```

Comment:
Should we use named parameters and naked return everywhere?
```
func (m *KeyValueStoreImplWithMetrics) ReadRange(ctx context.Context, table []byte, lkey Key, rkey Key) (it Iterator, err error) {
	m.measure("ReadRange", func() error {
		it, err = m.kv.ReadRange(ctx, table, lkey, rkey)
		return err
	})
	return
}
```

Reply:
Changed like this everywhere.